### ISSUE TITLE
Add support for authorizer_credentials in authorizers

### ISSUE THREAD

**Issue by dfradejas** [terraform-aws-modules/terraform-aws-apigateway-v2#70]
Hi,

We want to execute a Lambda function as authenticator for Api Gateway and we need permissions to execute it. For that, original resource has the parameter authorizer_credentials
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_authorizer#authorizer_credentials

We are checking the code and see that this parameter is not supported yet. Would you add this parameter for authorizers?

Regards,
Daniel

**Comment by dfradejas** [comment]
I made a fork and just adding this works.

```
diff --git a/main.tf b/main.tf
index 561beca..e677fa7 100644
--- a/main.tf
+++ b/main.tf
@@ -192,6 +192,7 @@ resource "aws_apigatewayv2_authorizer" "this" {
   authorizer_uri                    = try(each.value.authorizer_uri, null)
   authorizer_payload_format_version = try(each.value.authorizer_payload_format_version, null)
   authorizer_result_ttl_in_seconds  = try(each.value.authorizer_result_ttl_in_seconds, null)
+  authorizer_credentials_arn        = try(each.value.authorizer_credentials_arn, null)
 
   dynamic "jwt_configuration" {
     for_each = length(try(each.value.audience, [each.value.issuer], [])) > 0 ? [true] : []
```

Regards,
Daniel

**Comment by antonbabenko** [comment]
This issue has been resolved in version 1.8.0 :tada:
